lib/secret.ml

```diff
@@ -2,12 +2,12 @@ let __random_bytes bytes = Mirage_crypto_rng.generate bytes
 
 let generate ~bytes () =
   if bytes >= 10 && bytes mod 5 == 0
-  then Base32.string_to_base32 @@ Cstruct.to_string @@ __random_bytes bytes
+  then Base32.string_to_base32 @@ __random_bytes bytes
   else
     failwith
       ( "Invalid amount of bytes ("
       ^ string_of_int bytes
       ^ ") for secret, it must be at least 10 and divisible by 5!" )
 
 
-let _ = Mirage_crypto_rng_unix.initialize (module Mirage_crypto_rng.Fortuna)
+let _ = Mirage_crypto_rng_unix.use_default ()
```

Comment:
> mirage-crypto-rng: deprecate the initialize for lwt, async, eio (and
advertise Mirage_crypto_rng_unix.use_default () (https://github.com/mirage/mirage-crypto/pull/254 @hannesm)

Is this change ok  ?

Reply:
LGTM, `getentropy` is the way to go to ensure the system doesn't start running with low entropy at boot time